### DISCUSSION TITLE
Broken link on website and a suggestion for tip text

### DISCUSSION THREAD

**Discussion by instancezero** [vkbo/novelWriter#2041] [General]
My apologies if there's a better place to post this.

The link to Enchant on https://novelwriter.io/docs/more_customise.html produces a 404. The current link appears to be https://rrthomas.github.io/enchant/. It appears that they move the repo with the current maintainer, which is less than helpful.

I also suggest that in the preferences dialog, under "Text editing", the tip "Available languages are determined by your system." be replaced with "Additional languages can be added under Tools > Add dictionaries." since that dialog already has most of the information required to add a new dictionary.

**Comment by vkbo** [comment]
Thanks, I'll update the link.

The dictionaries dialog is Windows-only, so the text you mention is correct for Linux and MacOS.
